packages/ui-components/src/components/text-field/text-field.base.scss

```diff
@@ -40,9 +39,8 @@
 	--text-color-help-text-error: #{kv-color('error')};
 
 	--border-color-error: #{kv-color('error')};
-	--right-slot-padding-right: #{$spacing-9x};
 	--right-slot-width: 36px;
-	--left-slot-padding-left: #{$spacing-6x};
+	--left-slot-width: 32px;
```

Comment:
this is breaking, right?

Reply:
since it's a change to the name of this CSS prop, yes, it is breaking. when it comes to how this component is rendered, it should be the same :)

Reply:
added breaking notation on commit ;)